tutorials/nservicebus-101/lesson-3/tutorial.md

```diff
@@ -0,0 +1,203 @@
+
+![Exercise 3 Diagram](diagram.png)
+
+
+### Creating a new endpoint
+
+First, let's create the project for our new endpoint.
+
+ 1. Create a new **Console Application** project named **Sales**.
+ 1. In the **Sales** project, add the NServiceBus NuGet package:
+   ```no-highlight
+   Install-Package NServiceBus -ProjectName Sales
+   ```
+ 1. In the **Sales** project, add a reference to the **Messages** project, so that we have access to the `PlaceOrder` message.
+
+Now that we have a project for our Sales endpoint, we need to add similar code to configure and start an NServiceBus endpoint:
+
+snippet:SalesProgram
+
+Most of this configuration looks exactly the same as our ClientUI endpoint. It's critical for the configuration between endpoints to match (especially message transport and serializer) otherwise the endpoints would not be able to understand each other.
+
+For example, if the ClientUI endpoint used `.UseSerialization<XmlSerializer>()` while the Sales endpoint used `.UseSerialization<JsonSerializer>()`, the Sales endpoint would not be able to understand the XML-serialized messages it received from ClientUI, because it would be expecting JSON instead.
+
+While most of the configuration is the same, let me draw your attention to two specific lines that are different:
+
+snippet:EndpointDifferences
+
+The difference, of course, is the name "Sales" in the console title and `EndpointConfiguration` constructor, which defines the endpoint name for the Sales endpoint and gives it its own identity.
+
+This means that the Sales endpoint will create its own queue named `Sales` where it will listen for messages. We now have two processes that each have their own mailbox, so now we can send messages between them.
+
+NOTE: This is quite repetitive, but remember that this is still an introductory exercise. In the next course, we will explore methods that allow you to centralize most of this repetitive endpoint configuration code.
+
+
+### Debugging multiple projects
+
+At this point, we could run the Sales endpoint, although we wouldn't expect Sales to do anything except start up, create its queues, and then wait for messages that would never arrive. If you'd like, it's a good exercise to do, although you can skip it if you're in a hurry.
+
+What you'll find, however, is that in NServiceBus solutions it's common to want to run multiple projects (endpoints) at once. While you can right-click each project separately, and select **Debug** > **Start new instance** for each one, many developers who have never run into this requirement before are not aware that Visual Studio allows you to run multiple projects when you debug.
+
+To configure Visual Studio to start multiple projects when you start debugging:
```

Comment:
remove the below and link here https://msdn.microsoft.com/en-us/library/ms165413.aspx